packages/terra-form/tests/jest/Textarea.test.jsx

```diff
@@ -31,3 +31,39 @@ it('should render as controlled when just a default value is passed into the Tex
   const wrapper = mount(textarea);
   expect(wrapper).toMatchSnapshot();
 });
+
```

Comment:
Add prop tests for isInvalid & isAutoResizable.

Reply:
Added additional tests here. https://github.com/cerner/terra-core/pull/966/commits/c6ce06a08fd17d58b808d903dbd8bf19eafd06e1